### PR TITLE
add pslibrary

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/pslibrary/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,1240 @@
...
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Mg.rel-pz-spnl-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Hg.rel-pz-n-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Sb.rel-pz-n-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Tc.rel-pz-spn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/P.rel-pz-n-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Zr.rel-pz-spn-kjpaw_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Fe.rel-pz-spn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cl.rel-pz-nl-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Cr.rel-pz-spn-rrkjus_psl.1.0.0.UPF
+    - test -f ${PREFIX}/share/pslibrary/rel-pz/PSEUDOPOTENTIALS/Ce.rel-pz-spdfn-rrkjus_psl.1.0.0.UPF
```
Comment (ocefpaf) [inline recipes/pslibrary/meta.yaml:1221]: Can we be smarter about this and avoid a recipe with thousand of line? I don't like fancy jinjas and look but maybe this a case where we should use them.

Comment (jan-janssen) [reply recipes/pslibrary/meta.yaml:1221]: I could move the tests to a separate file, but I am not exactly sure how to solve it inside the yaml file with more fancy jinjas.

Comment (ocefpaf) [reply recipes/pslibrary/meta.yaml:1221]: To be honest I never wrote a jinja to do that, not even sure how to do it myself.
Moving to another file is a good compromise until we have a better solution.

File: recipes/pslibrary/tests.sh [hunk 2]
```diff
@@ -0,0 +1,1194 @@
+test -f ${PREFIX}/share/pslibrary/rel-pbe/PSEUDOPOTENTIALS/Ta.rel-pbe-spfn-rrkjus_psl.1.0.0.UPF
```
Comment (ocefpaf) [inline recipes/pslibrary/tests.sh]: Maybe here you can do a glob for all *.UPF, no?

Comment (jan-janssen) [reply recipes/pslibrary/tests.sh]: Does this help? I thought the idea is to confirm that all files are written. If I just use find or glob than it only identifies the ones which are written, but I do not notice missing ones.

Comment (ocefpaf) [reply recipes/pslibrary/tests.sh]: I'm not familiar with this package to be honest, so maybe my request does not make sense. Checking for each file is not sustainable and hard to maintain, but if it is necessary let's do it.

Comment (jan-janssen) [reply recipes/pslibrary/tests.sh]: It is similar like https://github.com/conda-forge/gpaw-data-feedstock and https://github.com/conda-forge/iprpy-data-feedstock

Comment (ocefpaf) [reply recipes/pslibrary/tests.sh]: My question is more like: "Is there a scenario where one of those files would not be created?"
If the expected result are all or nothing than a glob is fine. If the build can fail a single file and move on to a successful build, then there is not much we can do.

**Comment by jan-janssen** [comment]
Test seem to fail when only `run_test.sh` is used:
```
Traceback (most recent call last):
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 142, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 66, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 123, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3008, in build_tree
    packages_from_this = build(metadata, stats,
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2291, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1597, in bundle_conda
    output['checksums'] = create_info_files(metadata, replacements, files, prefix=metadata.config.host_prefix)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1253, in create_info_files
    create_all_test_files(m, test_dir=join(m.config.info_dir, 'test'))
  File "/opt/conda/lib/python3.8/site-packages/conda_build/create_test.py", line 260, in create_all_test_files
    shell_files = create_shell_files(m, test_dir)
  File "/opt/conda/lib/python3.8/site-packages/conda_build/create_test.py", line 80, in create_shell_files
    f.write(cmd)
TypeError: write() argument must be str, not bool
```

File: recipes/pslibrary/tests.sh (hunk 2 above)
Comment (jan-janssen) [reply recipes/pslibrary/tests.sh]: I guess for now I would test all and then update the tests as soon as I have some experience with the changes switching from one version to the next.

**Pull request merged**